<!--local position rectified by boundary detect-->
<launch>
   <node pkg = "iarc_tf" type = "ned_world_broadcaster_node" name ="ned_world_brocaster_node" output="screen" />
   <node pkg = "iarc_tf" type = "ned_world_transform_node" name = "ned_world_transform_node" output="screen"/>
</launch>
